main/EDMainMod.F90

```diff
@@ -27,6 +27,7 @@ module EDMainMod
   use FatesInterfaceTypesMod        , only : numpft
   use FatesInterfaceTypesMod        , only : hlm_use_nocomp
   use PRTGenericMod            , only : prt_carbon_allom_hyp
+  use PRTGenericMod            , only : prt_csimpler_allom_hyp
```

Comment:
@mpaiao was this intended to be used in this module?  It looks like its the only thing that was changed.

Reply:
@glemieux I was just tracking every place in the code where `prt_carbon_allom_hyp` appeared, and added this "use" statement to be consistent. But neither `prt_carbon_allom_hyp` nor `prt_csimpler_allom_hyp` are actually used in EDMainMod.F90.  Fine with me deleting it.